### ISSUE TITLE
panaroo-filter-pa

### ISSUE THREAD

**Issue by fabgenomics** [gtonkinhill/panaroo#148]
Hi,
I'm using panaroo-filter-pa script but the resulting file gene_presence_absence_filt_pseudo_length_frag.Rtab has a tab separated header and coma separated data.
Is it in the code or my pipeline is messing around with the file ?
Thanks for your help

**Comment by gtonkinhill** [comment]
Hi,

Thanks for pointing this out. It is a bug in the filter-pa script. I have just pushed a fix to the `devel` branch.
You should be able to update your installation using 

```
pip install git+https://github.com/gtonkinhill/panaroo.git@devel
```

I am hoping to publish a new release of panaroo with these updates in the next week or so.

**Comment by gtonkinhill** [comment]
This should now be fixed in v1.2.10
